frontend/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import MaterialTable from "material-table";
+import { Button, TableRow } from "@material-ui/core";
+
+function App() {
+
+
+  const [red, setRed] = useState(false);
+  const [green, setGreen] = useState(false);
+
+  return (
+    <div style={{ maxWidth: "100%" }}>
+      <MaterialTable
+        actions={[
+          {
+            icon: () => <Button variant='contained'>TOGGLE LOGGED IN </Button>,
+            tooltip: 'Filter',
+            onClick: () => setRed(!red),
+            isFreeAction: true
+          },
+          {
+            icon: () => <Button variant='contained' >TOGGLE NEVER LOGGED IN </Button>,
+            tooltip: 'Filter',
+            onClick: () => setGreen(!green),
```

Comment:
Whoops. I think you reversed the colors for the buttons here - should be `setRed(!red)`.